src/User/UserContainer.jsx

```diff
@@ -1,20 +1,29 @@
 import React, { useState, useEffect } from "react";
 import { Switch, Route, useParams } from "react-router-dom";
+import { useCookies } from "react-cookie";
+import { cookieName } from "../Constants/Cookies";
 import ProfilePage from "./ProfilePage";
-import { getUserById } from "../api.js";
 import UpdateProfile from "./UpdateProfile";
+import { getUserById, getFavouritedConferencesByUserId } from "../api.js";
 
-const UserContainer = ({ token }) => {
+const UserContainer = ({ sessionToken }) => {
```

Comment:
probably should remove this and get the cookie from the hook on line 15 to follow suit!